packages/ui-components/src/components/action-button/action-button.tsx

```diff
@@ -1,74 +1,56 @@
-import { Component, Event, EventEmitter, h, Prop, State } from '@stencil/core';
-import { isEmpty } from 'lodash-es';
-import { EActionButtonType } from './action-button.types';
+import { Component, Event, EventEmitter, h, Host, Prop } from '@stencil/core';
+import { EActionButtonSize, EActionButtonType } from './action-button.types';
```

Comment:
As discussed, please replace occurrences of `EActionButtonSize` with `EComponentSize` for consistency across component sizing definition.